src/Azure/Orleans.Clustering.AzureStorage/AzureBasedMembershipTable.cs

```diff
@@ -40,8 +39,13 @@ public async Task InitializeMembershipTable(bool tryInitTableVersion)
             AzureTableDefaultPolicies.MaxBusyRetries = options.MaxStorageBusyRetries;
             LogFormatter.SetExceptionDecoder(typeof(StorageException), AzureTableUtils.PrintStorageException);
 
-            tableManager = await OrleansSiloInstanceManager.GetManager(
-                this.clusterId, options.ConnectionString, options.TableName, this.loggerFactory);
+            this.tableManager = await OrleansSiloInstanceManager.GetManager(
+                this.clusterId,
+                this.options.ConnectionString,
+                this.options.TableName,
+                this.loggerFactory,
+                this.options.CreationTimeout,
```

Comment:
Could pass in options class rather than each specific option.  Class didn't originally because it was written before we started using options.